3p/3p.js

```diff
@@ -154,7 +154,7 @@ export function validateSrcPrefix(prefix, src) {
  * @param {string} src
```

Comment:
So everything in this file cannot be updated, because the 3p directory is run in foreign iframes. This means they will not have the `Array#includes` polyfill installed in them.